frontend/lib/src/components/widgets/TextInput/TextInput.tsx

```diff
@@ -15,7 +15,7 @@
  */
 
 import React from "react"
-import { uniqueId } from "lodash"
+import uniqueId from "lodash/uniqueId"
```

Comment:
From https://github.com/streamlit/streamlit/pull/7842 we want to import differently. I think we should add an eslint on this if possible.